docs/source/nas/exploration_strategy.rst

```diff
@@ -75,7 +75,14 @@ Starting from v2.8, the usage of one-shot strategies are much alike to multi-tri
 
    import nni.retiarii.strategy as strategy
    import nni.retiarii.evaluator.pytorch.lightning as pl
-   evaluator = pl.Classification(...)
+   evaluator = pl.Classification(
+     # Need to use pl.DataLoader instead of torch.utils.data.DataLoader here.
```

Comment:
might be better to addd one more sentence: users can refer to "serializer" doc if they stick to torch.utils.data.DataLoader